Source/Utilities/IsShortcutActive.lua

```diff
@@ -0,0 +1,21 @@
+function Auctionator.Utilities.IsShortcutActive(shortcutOption, button)
+  return (
```

Comment:
This is really involved, and its hard for me to understand all of the test cases is this form. Would it make sense to break them out into individual test cases, instead of having them together as a massive boolean statement? Even if only separating them into individual functions that could be chained, so it is a little easier to grok when reading...
```lua
function isLeftClickShortcutActive(shortcutOption, button)
  return 
    shortcutOption == Auctionator.Config.Shortcuts.LEFT_CLICK and
    button == LEFT_BUTTON -- constantizing to make it easier to avoid typos in strings
   -- etc.
end
```

Reply:
I can do that :)